CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 ### Upgrade notes
 
+- **Redis**
+
+Since this version `redis` gem is not a `decicim-core` dependency anymore. The PR intoducing this modification is [\#6084](https://github.com/decidim/decidim/pull/6084).
+
+If you are using `sidekiq` or other redis backed `active-job` backend make sure `redis` gem is still in your Gemfile.lock.
```

Comment:
This is not how dependencies work. Bundler always get it right, as `redis` is already a dependency for `sidekiq`. See https://rubygems.org/gems/sidekiq/versions/6.0.7/dependencies 

This message is not necessary at all

Reply:
```ruby
# Gemfile
source "https://rubygems.org"

gem "sidekiq"
```

```ruby
# Gemfile.lock
GEM
  remote: https://rubygems.org/
  specs:
    connection_pool (2.2.2)
    rack (2.2.2)
    rack-protection (2.0.8.1)
      rack
    redis (4.1.4)
    sidekiq (6.0.7)
      connection_pool (>= 2.2.2)
      rack (~> 2.0)
      rack-protection (>= 2.0.0)
      redis (>= 4.1.0)

PLATFORMS
  ruby

DEPENDENCIES
  sidekiq

BUNDLED WITH
   1.17.2


```